partner_shipping_policy/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    picking_policy = fields.Selection(
```

Comment:
I'm guessing the field could be added to the `_commercial_fields` method result? WDYT?

Reply:
Yes, I did not think of that but it makes sense. Thanks!